### DISCUSSION TITLE
What is control region?

### DISCUSSION THREAD

**Discussion by Rohit-Satyam** [broadinstitute/gatk#8195] [Q&A]
Hi Developers

Should we always use the following argument? What is control region? Shouldn't `-XL` be used to exclude region rather than `-L`?
```
-L chrM:576-16024
```

**Comment by meganshand** [comment]
@Rohit-Satyam There should be more information on this on the GATK forum: [Mitochondrial short variant discovery](https://gatk.broadinstitute.org/hc/en-us/articles/4403870837275-Mitochondrial-short-variant-discovery-SNVs-Indels-) If you have further questions about the [control region](https://en.wikipedia.org/wiki/MtDNA_control_region) and which arguments to use please feel free to post on the [GATK forum](https://gatk.broadinstitute.org/hc/en-us/community/topics).
